src/ntp.c

```diff
@@ -209,6 +211,17 @@ ntp_read_prefs (int soc)
       }
     }
 
+  if (buffer_size_count >= input_sz) /* finished = 1; */
+    {
+      log_write ("%s: The received plugin list is longer than expected.\n",
+                 __func__);
+    }
+
+#if DEBUGMORE
+  log_write ("%s: Received %d chars from the client as plugin list.\n",
+￼            __func__, buffer_size_count);
```

Comment:
Could it be possible that the line contains some strange character at the beginning?

Reply:
@bjoernricks,
I couldn't see the char, but removing all the spaces in that line and adding it again, there was a difference. So, pushing again with the invisible char.